content/cloud/faq.md

```diff
@@ -41,10 +41,12 @@ The following are the quota for users based, stratified by the three [Subscripti
 | Key value store value size (MB)* | 1 | 1 | Contact us
 | Key value store storage size (GB) | 1 | 2 | Contact us
 | Key value maximum keys* | 1,024 | 1,024 | Contact us
-| SQLite Storage | [See private beta documentation](/cloud/noops-sql-db.md) | [See private beta documentation](/cloud/noops-sql-db.md)  | [See private beta documentation](/cloud/noops-sql-db.md) 
+| SQLite Storage (GB) | 1 | 1 |  Contact us
 | **Regions** |
 | Region count| 1 | 1 | Contact us
 
+> Note: Storage quota is shared across all Fermyon Cloud objects (e.g., Developer Plan users have 1 GB of storage to share between their Key Value Stores and their NoOps SQL Databases; Growth Plan users have 2 GB of storage to share between their Key Value Stores and their NoOps SQL Databases)
```

Comment:
The free plan is called Starter in the table?

Reply:
Thanks @itowlson this has been updated in https://github.com/fermyon/developer/pull/938/commits/1a639ab2e47b8b634dda11a51ca032cb80c6f09e